osafw-app/App_Code/models/DBUpdates.cs

```diff
@@ -0,0 +1,65 @@
+// DBUpdates model class
+//
+// Part of ASP.NET osa framework  www.osalabs.com/osafw/asp.net
+// (c) 2009-2023 Oleg Savchuk www.osalabs.com
+
+using System.Collections;
+using System.ComponentModel.DataAnnotations;
+using System.Reflection;
+
+namespace osafw;
+
+public class DBUpdates : FwModel
+{
+    public const int STATUS_APPLIED = 50;
```

Comment:
why 50?
I can see the following statuses:
0 - not applied
10 - inactive (should not be applied, if for some reason we decided to skip it)
20 - attempt to apply - failed - idesc should contain and error
30 - applied successfully